src/lib/builders/slider/create.ts

```diff
@@ -194,13 +194,15 @@ export const createSlider = (props?: CreateSliderProps) => {
 				const $value = get(value);
 				const $orientation = get(orientation);
 
+				const $actual_max = add($min, (mul(Math.floor(div(sub($max, $min), $step)), $step)))
```

Comment:
Prefer `camelCase` syntax, otherwise we'll have conflicting cases.

Also, `add($min, (mul(Math.floor(div(sub($max, $min), $step)), $step)))` is really hard to parse. I'd suggest offloading the calculation into other constants/parts, and then joining them all together in a human-readable way.

Also, I assume we also would need an `actualMin` constant?

Reply:
I gave it a lot of thought if `actualMin` is relevant or not. At the moment the native range input doesn't do defaultValue like this. For example if you have min 0, step 3 and set value to 2, the actual value will be 3 from the start. For our case if we do that, the initialValue shows 2 but when we move it the ticks are always starting from min so it will be either 0 or 3. This is why `actualMin` is not relevant imho.

What is your suggestion for this case? 
Btw I am very new to os contributing so I'll take all the tips I can get :)

Reply:
> I gave it a lot of thought if `actualMin` is relevant or not. At the moment the native range input doesn't do defaultValue like this. For example if you have min 0, step 3 and set value to 2, the actual value will be 3 from the start. For our case if we do that, the initialValue shows 2 but when we move it the ticks are always starting from min so it will be either 0 or 3. This is why `actualMin` is not relevant imho.
> 
> What is your suggestion for this case? Btw I am very new to os contributing so I'll take all the tips I can get :)

I see! We should follow native behaviour IMO.